v2/yarpcjson/inbound.go

```diff
@@ -42,6 +42,15 @@ type jsonHandler struct {
 	handler reflect.Value
 }
 
+type jsonHandler2 struct {
+	handler reflect.Value
+}
+
+func (h jsonHandler2) Handle(ctx context.Context, reqBody interface{}) (interface{}, error) {
+	results := h.handler.Call([]reflect.Value{reflect.ValueOf(ctx), reflect.ValueOf(reqBody)})
```

Comment:
Let’s add some assertions here to ensure we don’t panic. Length of results. Conditionally cast second result to error.